functions/wait/action/kubernetesUpgrade.go

```diff
@@ -17,32 +17,44 @@ func KubernetesUpgrade(t *testing.T, client *rancher.Client, clusterID string, m
 
 	if module == "aks" || module == "gke" {
 		expectedUpgradedKubernetesVersion := `v` + clusterConfig.UpgradedKubernetesVersion
-		err := wait.Poll(100*time.Millisecond, 30*time.Minute, func() (done bool, err error) {
-			cluster, err := client.Management.Cluster.ByID(clusterID)
+		waiting := false
+		wait_err := wait.Poll(100*time.Millisecond, 30*time.Minute, func() (done bool, err error) {
```

Comment:
Overal LGTM, one nit would be snake case naming. https://go.dev/doc/effective_go#mixed-caps

Reply:
updated! thanks @brudnak